SlackChatbot/utils/config.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
Ensure that none of these environment variables (tokens/secrets) are committed in plaintext. Confirm that these values are only set via .env or environment variables and that .env is ignored